src/Umbraco.Web.UI.Client/src/common/services/tinymce.service.js

```diff
@@ -332,7 +347,35 @@ function tinyMceService($rootScope, $q, imageHelper, $locale, $http, $timeout, s
                     images_replace_blob_uris: false,
                     init_instance_callback: initEvents
                 };
-
+                
+                // We keep spans here, cause removing spans here also removes b-tags inside of them, instead we strip them out later.
+                var validElements = "-strong/b,-em/i,-u,-span,-p,-ol,-ul,-li,-p/div,-a[href|name],sub,sup,strike,br,del,table[width],tr,td[colspan|rowspan|width],th[colspan|rowspan|width],thead,tfoot,tbody,img[src|alt|width|height],ul,ol,li,hr,pre,dl,dt,figure,figcaption,wbr"
+                  
+                // add valid elements from styleFormats.
```

Comment:
Add a more detailed explanation of what you are trying to do here. Only allow a H3 to be pasted across from Word if you have it setup/configured as a user-defined rule in the CSS